pkg/cli/admin/release/extract_tools.go

```diff
@@ -1237,18 +1239,12 @@ func findClusterIncludeConfig(ctx context.Context, restConfig *rest.Config) (man
 	} else {
 		config.Overrides = clusterVersion.Spec.Overrides
 		config.Capabilities = &clusterVersion.Status.Capabilities
-
-		// FIXME: eventually pull in GetImplicitlyEnabledCapabilities from https://github.com/openshift/cluster-version-operator/blob/86e24d66119a73f50282b66a8d6f2e3518aa0e15/pkg/payload/payload.go#L237-L240 for cases where a minor update would implicitly enable some additional capabilities.  For now, 4.13 to 4.14 will always enable MachineAPI, ImageRegistry, etc..
-		currentVersion := clusterVersion.Status.Desired.Version
-		matches := regexp.MustCompile(`^(\d+[.]\d+)[.].*`).FindStringSubmatch(currentVersion)
-		if len(matches) < 2 {
-			return config, fmt.Errorf("failed to parse major.minor version from ClusterVersion status.desired.version %q", currentVersion)
-		} else if matches[1] == "4.13" {
-			build := configv1.ClusterVersionCapability("Build")
-			deploymentConfig := configv1.ClusterVersionCapability("DeploymentConfig")
-			imageRegistry := configv1.ClusterVersionCapability("ImageRegistry")
-			config.Capabilities.EnabledCapabilities = append(config.Capabilities.EnabledCapabilities, configv1.ClusterVersionCapabilityMachineAPI, build, deploymentConfig, imageRegistry)
-			config.Capabilities.KnownCapabilities = append(config.Capabilities.KnownCapabilities, configv1.ClusterVersionCapabilityMachineAPI, build, deploymentConfig, imageRegistry)
+		config.ClusterCapabilities = capability.SetCapabilities(clusterVersion, nil, nil)
+		for k := range config.ClusterCapabilities.KnownCapabilities {
+			config.Capabilities.KnownCapabilities = append(config.Capabilities.KnownCapabilities, k)
+		}
```

Comment:
With 1.23 you can now do
```suggestion
		config.Capabilities.KnownCapabilities = append(config.Capabilities.KnownCapabilities, slices.Collect(maps.Keys(config.ClusterCapabilities.KnownCapabilities))...)
```

and same thing for `EnabledCapabilities`